spec/controllers/my_contributions_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe MyContributionsController, type: :controller do
+  render_views
+
+  let(:user) { build_stubbed(:user) }
+  let(:contribution_list) { [build_stubbed(:contribution), build_stubbed(:contribution)] }
+  let(:contribution_model) { double('ActiveRecord', order: contribution_list) }
+  let(:page) { Capybara::Node::Simple.new(response.body) }
+
+  before do
+    allow(controller).to receive(:authenticate_user!)
+    allow(controller).to receive(:current_user).and_return(user)
+    allow(user).to receive(:contributions).and_return(contribution_model)
+    allow(contribution_model).to receive(:where).and_return(contribution_model)
+  end
+
+  describe 'GET index' do
+    it 'renders the index template' do
+      get :index
+
+      expect(response).to render_template(:index)
+    end
+
+    it 'has status 200' do
+      get :index
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    context 'when the user does not contain contributions' do
+      let(:contribution_list) { [] }
+
+      it 'renders the table with not found message' do
+        get :index
+
+        expect(page.find('table')).to have_content(I18n.t('my_contributions.no_prs_found'))
+      end
+    end
+
+    context 'when the user contains contributions' do
+      it 'renders the table with a list of contributions' do
+        get :index
+
+        expect(page.find('table')).to have_content(contribution_list[0].link)
+                                  .and have_content(Contribution.human_attribute_name("pr_state/#{contribution_list[0].pr_state}"))
```

Comment:
```suggestion
                                  .and have_content(contribution_list[0].pr_state_text"))
```
The same approach can be applied here

Reply:
Fixed